tldw_Server_API/app/core/AuthNZ/federation/claim_mapping.py

```
@@ -0,0 +1,93 @@
+        else:
+            values.append(mapped)
+    return values
+
+
+def _coerce_int_list(values: list[Any]) -> list[int]:
+    result: list[int] = []
+    for value in values:
+        try:
+            result.append(int(value))
+        except (TypeError, ValueError):
+            continue
```

Ruff (0.15.6)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)